### ISSUE TITLE
My Account section tab order

### ISSUE THREAD

**Issue by soumya-ashok** [magento-research/pwa-studio#1470]
Review and tweak keyboard tab order for the Venia 'My Account' section

**Comment by dani97** [comment]
@magento I am working on this

**Comment by sirugh** [comment]
This ticket was being worked on but the original contributor had other demands of their time so for now it is moving to the backlog. 

Whomever wishes to pick this up, please reference existing the existing PR, #2822, that may contain relevant context about the desired solution.

**Comment by sirugh** [comment]
@magento export issue to Jira project PWA as Story

**Comment by magento-engcom-team** [comment]
:white_check_mark: Jira issue (https://jira.corp.magento.com/browse/PWA-1278) is successfully created for this issue.

**Comment by vasilii-b** [comment]
Hi @sirugh , @soumya-ashok,
I wonder if this issue is still actual and whenever the changes made for #1222 covers what's required here.
Isn't "Left Nav" = "Filters"? 🤔 
Looking forward to your thoughts.

**Comment by sirugh** [comment]
You're right - this may not be necessary anymore since we implemented Mega Menu and no longer show left nav on desktop. Let me double check with the team :)

**Comment by pankhurigoel** [comment]
@magento I am working on this

**Comment by pankhurigoel** [comment]
@magento add to contributors team

**Comment by pankhurigoel** [comment]
@magento I am working on this
